src/skills-builder/skills-builder-modal/view-results/ViewResults.jsx

```diff
@@ -106,7 +135,8 @@ const ViewResults = () => {
         <RelatedSkillsSelectableBoxSet
           jobSkillsList={jobSkillsList}
           selectedJobTitle={selectedJobTitle}
-          onChange={(e) => setSelectedJobTitle(e.target.value)}
+          // TODO: add handle function to call setSelectedJobTitle and the 'shown' event
```

Comment:
Do we still need this TODO? It looks like we may have implemented it?

Reply:
Good catch!